4.6.2/Dockerfile

```diff
@@ -9,6 +9,9 @@ RUN Add-WindowsFeature Web-Server; \
 
 ADD ServiceMonitor.exe /
 
+ADD ConfigureRoslynCompilers.ps1 .
```

Comment:
I do not believe encapsulating this logic in a separate script is appropriate - https://github.com/docker-library/official-images/blob/master/README.md#clarity.  I do not think this would pass an official-image review which is the bar we should be holding ourselves to.

Reply:
See also https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-docker/optimize-windows-dockerfile for some tips on how to put the script in the dockerfile while avoiding zillions of RUN statements.